trajopt_optimizers/trajopt_optimizers_test_suite/src/run_tests.cpp

```diff
@@ -0,0 +1,11 @@
+#include <trajopt_utils/macros.h>
+#include <gtest/gtest.h>
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+  std::cout << "testing" << std::endl;
+  testing::InitGoogleTest(&argc, argv);
+
+  return RUN_ALL_TESTS();
+}
```

Comment:
Just put this at the bottom of each test file like we do [here](https://github.com/ros-industrial-consortium/trajopt_ros/blob/9a85cd1800d9129ff9d46734f5a13c721b66065e/trajopt_ifopt/test/joint_terms_unit.cpp#L42)